README.md

```diff
@@ -1,119 +1,11 @@
-# Prometheus
+# Prometheus with Uyuni Service Discovery
 
-[![CircleCI](https://circleci.com/gh/prometheus/prometheus/tree/master.svg?style=shield)][circleci]
-[![Docker Repository on Quay](https://quay.io/repository/prometheus/prometheus/status)][quay]
-[![Docker Pulls](https://img.shields.io/docker/pulls/prom/prometheus.svg?maxAge=604800)][hub]
-[![Go Report Card](https://goreportcard.com/badge/github.com/prometheus/prometheus)](https://goreportcard.com/report/github.com/prometheus/prometheus)
-[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/486/badge)](https://bestpractices.coreinfrastructure.org/projects/486)
-[![fuzzit](https://app.fuzzit.dev/badge?org_id=prometheus&branch=master)](https://fuzzit.dev)
-[![Gitpod ready-to-code](https://img.shields.io/badge/Gitpod-ready--to--code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/prometheus/prometheus)
+This is a fork of Prometheus with the purpose of maintaining Uyuni service discovery.
```

Comment:
If the objective is to upstream, maybe we want to remove this, or even revert file to the original?

Reply:
`uyuni-2.22` branch is to bring SD up-to-date with current Prometheus 2.22 version and as base for SUSE Prometheus package.
After merging this on I'll create a new branch rebased on `master` and drop the README commit.